src/tpi/data.rs

```diff
@@ -640,7 +662,40 @@ typedef enum CV_ptrtype_e {
     CV_PTR_64           = 0x0c, // 64 bit pointer
     CV_PTR_UNUSEDPTR    = 0x0d  // first unused pointer type
 } CV_ptrtype_e;
+*/
+
+/// The kind of a `PointerType`.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum PointerKind {
+    /// 16 bit pointer.
+    Near16,
+    /// 16:16 far pointer.
+    Far16,
+    /// 16:16 huge pointer.
+    Huge16,
+    /// Based on segment.
+    BaseSeg,
+    /// Based on value of base.
+    BaseVal,
+    /// Based on segment value of base.
+    BaseSegVal,
+    /// Based on address of base.
+    BaseAddr,
+    /// Based on segment address of base.
+    BaseSegAddr,
+    /// Based on type.
+    BaseType,
+    /// Based on self.
+    BaseSelf,
+    /// 32 bit pointer.
+    Near32,
+    /// 16:32 pointer.
+    Far32,
+    /// 64 bit pointer.
+    Ptr64,
+}
 
+/*
```

Comment:
Please delete this now that you implemented `PointerMode`.

Reply:
Removed.